src/backend/src/services/projects.services.ts

```diff
@@ -605,6 +606,58 @@ export default class ProjectsService {
   }
 
   /**
+   * Create an assembly
+   * @param name The name of the assembly to be created
+   * @param pdmFileName The name of the file holding the assembly
+   * @param userCreated The user creating the assembly
+   * @param wbsElementId The
+   * @returns the project that the user has favorited/unfavorited
+   * @throws if the project wbs doesn't exist or is not corresponding to a project
+   */
+  static async createAssembly(
+    name: string,
+    pdmFileName: string | '',
```

Comment:
you can do pdmFileName?: string to make this optional, and then (once the schema is fixed), you should be able to just pass it directly into the prisma call